user-interface/src/main/java/life/qbic/datamanager/Application.java

```diff
@@ -42,8 +41,7 @@
     shortName = "Data Manager",
     offlineResources = {"images/logo.png"})
 @NpmPackage(value = "line-awesome", version = "1.3.0")
-@ComponentScan({"life.qbic"})
-@EntityScan(basePackages = "life.qbic")
```

Comment:
so this was preventing the data sources from being found?

Reply:
Yes. It created the datasources and injected them prior to the specific configurations taking place.